src/JSInterop/Microsoft.JSInterop/src/Infrastructure/DotNetInvocationResult.cs

```diff
@@ -12,9 +12,9 @@ namespace Microsoft.JSInterop.Infrastructure
         /// </summary>
         /// <param name="exception">The <see cref="System.Exception"/> that caused the failure.</param>
         /// <param name="errorKind">The error kind.</param>
-        public DotNetInvocationResult(Exception exception, string? errorKind)
+        internal DotNetInvocationResult(Exception exception, string? errorKind)
```

Comment:
How do you feel about leaving the API on this type as-is but interpreting `Result` as a string value?

Reply:
TBH I'd rather take the opportunity to clean things up as much as possible if we're taking a breaking change anyway.